migration/v0.46/apply/30-migration-alertmanager.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+export CK8S_STACK="migration/user-alertmanager-secret"
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+source "${ROOT}/scripts/migration/lib.sh"
+
+# Ensure wc config is loaded
+config_load wc
+
+# Extract alertmanager config from old secret
+kubectl_do wc get secret alertmanager-alertmanager -n alertmanager -o jsonpath='{.data.alertmanager\.yaml}' | base64 -d >alertmanager.yaml
+
+if [[ ! -s alertmanager.yaml ]]; then
+  log_fatal "Extracted alertmanager.yaml is empty or missing"
+fi
+
+# Patch the new secret using wc kubeconfig
+kubectl_do wc patch secret alertmanager-kube-prometheus-stack-alertmanager -n alertmanager -p "{\"data\":{\"alertmanager.yaml\":\"$(base64 -w 0 <alertmanager.yaml)\"}}"
```

Comment:
I might be missing something, but it looks like this will not be created yet. Because I do not see any command here to install the new alertmanager release.
Unless I'm missing something I suggest you add a command in this script to install the new alertmanager helm release.